src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objValueOfDate = {};
+  const arrDateData = date.split(fromFormat[3]);
```

Comment:
Consider destructuring arrDateData directly from date.split for cleaner code. For example: const [day, month, year] = date.split(fromFormat[3]);

Reply:
but I don't understand how I can use destructuring in this case